app/src/org/commcare/views/widgets/QuestionWidget.java

```diff
@@ -135,6 +135,9 @@ public QuestionWidget(Context context, FormEntryPrompt p, boolean inCompactGroup
     protected void acceptFocus() {
     }
 
+    public void removeFocus() {
```

Comment:
Since this is only ever meant for `StringWidget` we should not include it here. Instead when you are calling this methos, you can do a type check and cast to call this only on `StringWidget`